episodes/parallel.Rmd

```diff
@@ -151,62 +112,27 @@ Add these functions to `functions.R` (you can copy-paste the original ones, then
 Then, change the plan to use the "slow" version of the functions:
 
 ```{r}
-#| label: example-model-show-9
-#| eval: FALSE
-source("R/packages.R")
-source("R/functions.R")
-
-library(crew)
-tar_option_set(
-  controller = crew_controller_local(workers = 2)
-)
-
-tar_plan(
-  # Load raw data
-  tar_file_read(
-    penguins_data_raw,
-    path_to_file("penguins_raw.csv"),
-    read_csv(!!.x, show_col_types = FALSE)
-  ),
-  # Clean data
-  penguins_data = clean_penguin_data(penguins_data_raw),
-  # Build models
-  models = list(
-    combined_model = lm(
-      bill_depth_mm ~ bill_length_mm, data = penguins_data),
-    species_model = lm(
-      bill_depth_mm ~ bill_length_mm + species, data = penguins_data),
-    interaction_model = lm(
-      bill_depth_mm ~ bill_length_mm * species, data = penguins_data)
-  ),
-  # Get model summaries
-  tar_target(
-    model_summaries,
-    glance_with_mod_name_slow(models),
-    pattern = map(models)
-  ),
-  # Get model predictions
-  tar_target(
-    model_predictions,
-    augment_with_mod_name_slow(models),
-    pattern = map(models)
-  )
-)
+#| label = "example-model-show-9",
+#| eval = FALSE,
+#| code = readLines("files/plans/plan_10.R")[3:42]
 ```
 
 Finally, run the pipeline with `tar_make()` as normal.
 
 ```{r}
 #| label: example-model-hide-9
-#| echo: FALSE
-tar_dir({
-  # Non-slow functions
-  write_example_plan(file = "plan_7.R")
-  tar_make(reporter = "silent")
-  # Slow functions
-  write_example_plan(chunk = "example-model-show-9")
-  tar_make()
-})
+#| warning: false
+#| message: false
+#| echo: false
+plan_10_dir <- make_tempdir()
+pushd(plan_10_dir)
+# Non-slow functions
+write_example_plan("plan_9.R")
+tar_make(reporter = "silent")
+# Slow functions
+write_example_plan("plan_10.R")
+tar_make()
+popd()
```

Comment:
fyi you don't need to use `pushd` and `popd` when the whole chunk is hidden, `tar_dir()` etc would be fine. I added them only for the scenario where we want to show *part* of the chunk and hide the directory manipulation, using `echo: [X, Y]`. However if you want to use `pushd` and `popd` universally for consistency, that's fine too.

Reply:
Yes, this was for consistency. Leaving as-is.